root/release/edit/tracklist.tt

```diff
@@ -280,11 +280,15 @@
         <td class="format">
           <label data-bind="attr: { for: 'disc-format-' + uniqueID }">[% l('Format:') %]</label>
           <select data-bind="value: formatID, attr: { id: 'disc-format-' + uniqueID }">
-            <option value="" selected="selected">&#160;</option>
+            <option value="" selected="selected">--</option>
```

Comment:
I have no opinion on the "no value" text but would like to point out that every other input on the website uses a blank option (see the information tab for example :)) so there is an inconsistency. Though it's out of the scope of this PR to change all of them, and I don't know what the benefit would be.

Reply:
I do agree with that, but since @chhavijustme asked for it, I did it. It's a good start, and it's not too terrible of an inconsistency so as to confuse users.

Reply:
We are disabling checkbox if some option is selected. In case you don't know the format, one has to unselect the option, to select the checkbox. In which case, you are to choose the "no value text", which is not very evident.  I expect an "*It's not letting me unselect an option to select the checkbox*" case. Hence the suggestion to use "--" .

@mwiencek  @yvanzo You can take a call on if we want to start using this now or later to keep up with the consistency. I feel we will have to address this someday for sure though :)

Reply:
I'm okay with updating them incrementally if you think it's a good idea, I don't think anyone will be confused.

Not to bikeshed here, but would it make sense to use an em dash (—) instead of two hyphens (--) since we prefer Unicode punctuation?

Reply:
Yes. em dash should work equally fine.

Reply:
Updated.